docs/api_overview.rst

```diff
@@ -361,12 +361,13 @@ To search for recent Sentinel 2 imagery by MGRS tile, use the `tileid` parameter
 
   api.download_all(products)
 
-NB: The `tileid` parameter may be missing from the metadata in SciHub's DHuS catalogue,
-in particular for older products. To be on the safe side, combine the `tileid` search
-with a `filename` pattern search:
+NB: The `tileid` parameter may not be available for all products, in which case 
+the products simply are not found. On the Copernicus Open Access Hub,
+you can use it to find Sentinel-2 L1C products from recent years (product type S2MSI1C).
```

Comment:
Can we point to any resource that helps users understand _how recent_?

Reply:
I think this is neither systematic nor documented. Perhaps I should write that.